Applications/CausalLM/layers/qwen_moe_layer_cached.cpp

```diff
@@ -313,24 +313,10 @@ void CachedSlimMoELayer::incremental_forwarding(
     nntrainer::Tensor &gate_weights = context.getWeight(gate_idx);
     input.dot(gate_weights, router_logits);
     router_logits.apply(nntrainer::ActiFunc::softmax<float>, router_logits);
+    const int alpha = 2; // number of expert to be additionally cached
```

Comment:
not now, but it can be configurable.
And also, this moe layer should be support default Experts as in Deepseek.